### ISSUE TITLE
Improve documentation for `Linearity`

### ISSUE THREAD

**Issue by hrzhuang** [JacquesCarette/Drasil#3449]
Context: #3448

**Comment by daijingz** [comment]
@hrzhuang Can you give me more information?

**Comment by daijingz** [comment]
Which documentations am I required to change?

**Comment by samm82** [comment]
The issue title seems to be a bit misleading; I think the actual task here is that `Linearity` (see below) should be changed to something that makes more sense (following the linked issue), including its documentation (the comment above its definition).

https://github.com/JacquesCarette/Drasil/blob/d5afe5c61d5fac443a5eac96fef5856bf07454e8/code/drasil-printers/lib/Language/Drasil/Plain/Print.hs#L28-L29

**Comment by daijingz** [comment]
@samm82 In this issue, I feel I only need to change documentation (nothing needed to change with the code) Is this correct?

**Comment by daijingz** [comment]
@samm82 So there are two parts of work in this issue: change the code and the documentation.
For the code I am going to replace the definition name with this: data SingleLine = OneLine | MultiLine, while its corresponding comments need to be changed too. (Some of them still have references with the name "Linearity")
![U`HN$99)(1CNTH@_K_O5P2I](https://github.com/JacquesCarette/Drasil/assets/61481741/366cfd6c-540e-4659-8cbb-43c523f976c8)

![({SK4U1 TOEZK4B`EQ96TRW](https://github.com/JacquesCarette/Drasil/assets/61481741/2f3ecd15-7c09-4209-ada8-ee48a94a0905)

**Comment by daijingz** [comment]
@samm82 For the second part (documentation), where is the corresponding documentation position for `Linearity`? I searched all documents in SRS, but I feel there are no occurrences of the word "Linearity". (Not only in the SRS, it does not occur elsewhere)

As you can see the evidence below, only one file contains "Linearity", and this file is not a documentation:
![CC$SW8$TT(SA(VY JZ9Y7$3](https://github.com/JacquesCarette/Drasil/assets/61481741/93056268-5a4d-4bf4-84d8-3241af82e8c7)

**Comment by samm82** [comment]
Right, documentation can refer to "generated documentation for our code examples" **OR** "manual documentation for our Drasil code". This specific usage refers to the second, not the first; updating the comment above the definition of the new `data SingleLine` is what we're looking for
